DataMgr/ForeignStorage/ArrowCsvForeignStorage.cpp

```diff
@@ -201,7 +201,6 @@ void ArrowCsvForeignStorage::prepareTable(const int db_id,
   td.hasDeletedCol = false;
 }
 
-// TODO: optimize the number of arguments
```

Comment:
I haven't found any way to optimize the number of arguments here, but I could do this for ForeignStorageInterface::registerTable.